molecule/default/molecule.yml

```diff
@@ -53,6 +54,14 @@ platforms:
       - ${MOLECULE_PROJECT_DIRECTORY}/target:/var/tmp/target
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     privileged: true
+  # rpm
+  - name: almalinux-8
```

Comment:
aren't these essentially the same thing? do we get enough benefit for the 'cost' of running?

(non blocking)

Reply:
I don't have a dog in the Rocky vs Alma race.